### PR TITLE
Setup couch db

### PR DESCRIPTION
**Pull request by amosNash**
Npm run seed 

• Refactored seeding script 
• Set up couchDB
• Seeded 10Million records into database

### CODE REVIEW

File: show-memory-limit.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const v8 = require('v8');
+
+console.log(v8.getHeapStatistics());
```
Comment (christulin) [inline show-memory-limit.js:5]: never seen this before, but looks useful

File: server/index.js [hunk 2]
```diff
@@ -14,6 +15,14 @@ app.use(bodyParser.json());
 app.use(cors());
 app.use('/rooms/:id', express.static(__dirname + '/../client/dist'));
 
+
+
+app.get('/', async(req, res) => {
+  couch.insertIntoCouchDB({"stayId": 14, "numBed": 5}, (status) => {
+    res.sendStatus(status);
+  });
+})
```
Comment (christulin) [inline server/index.js]: Is there a reason this is the root route? Or is this for testing purposes

File: package.json [hunk 3]
```diff
@@ -51,6 +52,8 @@
     "mongoose": "^4.9.6",
     "msw": "^0.25.0",
     "multer": "^1.4.2",
+    "nano": "^9.0.3",
+    "node-couchdb": "^1.3.0",
```
Comment (christulin) [inline package.json:56]: are you using this `node-couchdb` package?

File: database/data.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
+   await seedDB(1000000, testcouch);
```
Comment (christulin) [inline database/data.js]: Wondering if you found this was more memory efficient than just passing in 10000000 to `seedDB`.. also, might be cleaner to turn this into a loop vs 10 repeated lines

**Review by christulin — APPROVED** [review]
Looks good!

File: server/index.js (hunk 2 above)
Comment (amosNash) [reply server/index.js]: Ya this was for testing purposes. I should take it out.

File: package.json (hunk 3 above)
Comment (amosNash) [reply package.json:56]: I was at first but then switched to nano, so I should uninstall it.

File: database/data.js (hunk 4 above)
Comment (amosNash) [reply database/data.js]: I think thats a good idea to do a loop. I'm not sure why but I wasn't able to directly pass in 10000000 as I would get an error that memory heap is full. When I did it this way it worked out... I'm going to try and make it handle a single pass but for now this was my best solution.

File: show-memory-limit.js (hunk 1 above)
Comment (amosNash) [reply show-memory-limit.js:5]: Yeah I was just curious how much memory I was using.

**Pull request merged**